### DISCUSSION TITLE
update version to 0.13.3

### DISCUSSION THREAD

**Discussion by Egor1408** [xmppjs/xmpp.js#1035] [Q&A] [answered]
hello, today a version update was released, but in NPMJS not all versions are updated, which is why the installation of packages fails
![image](https://github.com/user-attachments/assets/accd8866-3571-4491-a453-2548dfcd7a1c)

![image](https://github.com/user-attachments/assets/36616d11-62cf-4c88-8b50-79b230e026ff)

![image](https://github.com/user-attachments/assets/04def2f7-c056-465f-a8b1-e4d77c16ae16)

**Comment by sonnyp** [comment, marked as answer]
Fixed.

**Reply by Egor1408** [reply]
thanks a lot
